docs/sphinx/dev_guide/kernel_class_impl.rst

```diff
@@ -60,6 +65,17 @@ The methods in the source file are:
         not every kernel implements every variant, so ``KernelBase`` provides a
         "default" implementation that defines no variants or tunings.
 
+    ..note:: When counting the number of bytes accessed each byte accessed is
+             counted in exactly one of the byte counters. For example a byte
+             that read and written is counted in the bytes read, modified, and
+             written counter and not in the read or written counters. Each byte
```

Comment:
```suggestion
             that is read and written is counted in the bytes read, modified, and
             written counter and not in the read or written counters. Each byte
```

This is confusing? If I read and write some chunk of bytes, why would it not be counted once in the read counter adn once in the write counter?

Reply:
Perhaps on the user output side we could include the RMW bytes with the R bytes and the W bytes, but on the input side its easier to separate things into mutually exclusive categories, only have to list it once instead of 1 times.

Reply:
I'll add the RMW bytes into the R and W bytes for user outputs. But I think its easier to count them into mutually exclusive buckets when developing/adding a kernel..